pandas/tests/extension/test_ops.py

```diff
@@ -0,0 +1,22 @@
+import pytest
```

Comment:
I didn't see a good place for generic ops tests. Am I missing someplace?

Reply:
You mean one that is not supposed to be subclassed by the actual EA implementations?

Reply:
I mean things that are testing pandas dispatching, not something in pandas (so not in tests/arrays) and not something that's part of the EA interface (so not part of tests/extension/base).

Reply:
> not something that's part of the EA interface (so not part of tests/extension/base).

OK, but basically we have been testing the ScalarOpsMixin through its use in DecimalArray I think, so you could see this one also as such a test.